Automattic/tumblr-theme-garden
# Add Tumblr audio block render

### Summary

Added rendering of the extended tumblr audio block

### Solution

Changes the html when rendering the audio block, see https://developer.wordpress.org/news/2024/08/15/how-to-extend-a-wordpress-block/

### Testing steps

1. Add an audio block to a post
2. Add custom attributes mediaTitle, mediaAlbum, mediaArtist
3. Verify attributes are set with code view in the editor
4. Publish and view post
5. It should render with tumblr markup

T3 render:
<img width="630" alt="Screenshot 2024-11-29 at 13 26 07" src="https://github.com/user-attachments/assets/fae5ff04-708a-40e4-bac4-2369f68470c0">

Tumblr rendered:
<img width="568" alt="Screenshot 2024-11-29 at 13 26 26" src="https://github.com/user-attachments/assets/20fe3055-11fe-4bc5-945f-e4b8ac518621">

### Reviewing tips

🧁 Request a review to team CupcakeLabs to assign a random team member.

🔄 Feel free to ping the team name again to "re-roll" or add another team member.

🚀 For now there is no requirement on getting review approvals before merging.
